sklearn/ensemble/tests/test_forest.py

```diff
@@ -197,7 +196,11 @@ def test_probability():
         yield check_probability, name
 
 
-def check_importances(name, criterion, X, y):
+def check_importances(name, criterion, X, y, dtype):
```

Comment:
It's just that the arrays are long and the end of the parameters are truncated. Putting dtype before X would render the report more informative. The actual content of X and y is not very informative.